app/src/substrate/substrate_coin.h

```diff
@@ -50,20 +50,16 @@ typedef enum {
 
 } key_kind_e;
 
-// Zondax major delta to fit chain tx version into the major value.
-// Major - this delta -> results on the chain tx version
-#define ZX_MAJOR_VERSION_DELTA 100
```

Comment:
Any reason why we removed this? Now the 100 is meaningless from the code's point of view.